ledger-bridge.js

```diff
@@ -260,9 +279,67 @@ export default class LedgerBridge {
 
         } finally {
             this.cleanUp()
+            this.clearPollingInterval()
         }
     }
 
+    onConnect() {
+        const pollConnection = async () => {
```

Comment:
Note here on the polling:

I connected the app by having the Ledger unlocked and Ethereum app ready , then exited the Ethereum app, and the disconnect event fired -- great!  ... but I then connected with the bitcoin app open and we got a positive/green message that the Ledger was ready, which is incorrect.